### ISSUE TITLE
Running NVDA from source in the user's profile directory may fail on newer Windows 11 installations

### ISSUE THREAD

**Issue by bramd** [nvaccess/nvda#19189]
### Related issues, PRs or discussions

Couldn't find any existing reports about this

### What is the current state of the codebase?

It seems newer Windows 11 installations have tighter security settings. I was testing NVDA on a clean Windows 11 virtual machine. Installed the machine with 24H2 and upgraded to 25H2 afterwards. Running from source gave:

```
core failure
Traceback (most recent call last):
File "nvda.pyw", line 309, in <module>
core.main()
~~~~~~~~~^^
File "core.py", line 719, in main
NVDAHelper.initialize()
~~~~~~~~~~~~~~~~~~~~~^^
File "NVDAHelper\__init_\_.py", line 851, in initialize
_remoteLoaderX86 = _RemoteLoader(ReadPaths.versionedLibX86Path)
File "NVDAHelper\__init__.py", line 752, in __init__
winKernel.CreateProcessAsUser(token, None, loaderPath, None, None, True, 0, None, None, si, pi)
~~~~~~~~~~~~~~~~~~~~~~~~~~~~~^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
File "winKernel.py", line 462, in CreateProcessAsUser
raise WinError()
OSError: [WinError 193] %1 is not a valid Win32 application.
```

Putting the source clone outside the user's profile directory allowed me to run it just fine. However, discovering what was going on took me quite some time.

### Why are changes required?

Many contributors might clone somewhere in their profile directory and may not expect the issue is Windows security related based on the exception and since this just worked fine in the past.

### What technical changes are required?

Either catch this  exception and give the user a hint what may be going on, or investigate this further and document how to prevent it.

Things I tried and didn't work:

* Disabled Windows Application Control
* Added an exception for the clone directory in Windows Defender

### Are the proposed technical changes API breaking?

No

### Are there potential risks or issues with the proposed implementation?

No

**Comment by LeonarddeR** [comment]
Have you also investigated [administrator protection](https://blogs.windows.com/windowsdeveloper/2025/05/19/enhance-your-application-security-with-administrator-protection/)? I believe parts of this are already available in released Windows. This issue also makes me think about https://github.com/nvaccess/nvda/issues/18955. It might also be interesting to try to run the source copy as admin and see what that does.

**Comment by bramd** [comment]
@LeonarddeR I can confirm that it also fails in the same way when running as an administrator, tried this using sudo in Powershell.

**Comment by seanbudd** [comment]
Is this in a directory which OneDrive syncs? There might be issues related to locking/syncing when trying to run

**Comment by bramd** [comment]
@seanbudd Should be, this is a clean VM with only a local user account. Since I never logged in with a Microsoft account anywhere, I'm sure OneDrive is not syncing anything.

**Comment by seanbudd** [comment]
Is this still something you are experiencing?

**Comment by bramd** [comment]
@seanbudd Hard to say. I created a dev drive on the machine with the problem and that is working fine. Didn't spin up any new/clean Windows machines after that. So I have no new concrete info here.

For me, it's fine to close this issue for now. If other people experience this they can at least find this and reopen/comment.
